lib/css/CssParser.js

```diff
@@ -278,10 +278,7 @@ class CssParser extends Parser {
 				dep.setLoc(sl, sc, el, ec);
 				module.addDependency(dep);
 				declaredCssVariables.add(name);
-			} else if (
-				propertyName.toLowerCase() === "animation-name" ||
-				propertyName.toLowerCase() === "animation"
-			) {
+			} else if (/^(-\w+-)?animation(-name)?$/i.test(propertyName)) {
```

Comment:
Let's hoist these and give them useful names. Even if they are one-of-a-kind match instances, this will make it easier to reason about. 

perhaps `OPTIONALLY_VENDOR_PREFIXED_ANIMATION_TAG_REGEXP` is a bit too verbose but something in between. 😅